src/components/EventMap.js

```diff
@@ -76,6 +76,13 @@ export default function(store){
         if (newFilters.zipcode !== oldFilters.zipcode) {
           this.setMapPositionBasedOnZip();
         }
+      },
+
+      // when the map is toggled to, make sure to resize accordingly
+      view(newView, oldView) {
+        if (newView === 'map' && this.mapRef) {
```

Comment:
Good q! I was kind of pretending that vue would only execute this function when the value does change, but I'm not sure if that is correct. I'll add that check.